src/components/ebay-textbox/template.marko

```diff
@@ -1,4 +1,81 @@
+<if(data.initIcons)>
+<span style="display: none">
+        <ebay-icon type="inline" name="add"/>
+        <ebay-icon type="inline" name="arrow-move" if(data.someFalsyProperty)/>
```

Comment:
@rachelvillamin Rather than adding markup for every icon name, you can maintain an array and loop it to create multiple icons.
lets say
```
<for (name in iconnames_array)>
   <ebay-icon type="inline" name="${name}" />
</for>
```
Something like this wouldn't bloat the template.

Reply:
I don't think this will work as the icon needs to be stamped initially as per ebay-icon documentation. See my comment above. And also screenshot below:
![screen shot 2018-11-15 at 11 10 29 am](https://user-images.githubusercontent.com/20077653/48521063-2283d080-e8c7-11e8-87b5-5138fffbf163.png)

Reply:
All of the approaches mentioned so far will cause all icons to be shipped to the browser in the JS bundle. We need to use a transformer to avoid that, similar to what we did for the `ebay-menu` icon. https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-menu/transformer.js

Reply:
Thanks Dylan. I will use a transformer. Declaring the ebay icons explicitly was the only way I could get it to work yesterday. I didn't know about transformers :) Thanks for your help!

Reply:
hi @DylanPiercey,
I have implemented rendering the icons using transformer. It's working. Not sure if I properly implemented it though :P Need your feedback. Thanks!